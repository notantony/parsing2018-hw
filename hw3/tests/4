f = if true print true print false