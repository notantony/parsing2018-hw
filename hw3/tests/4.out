void f() {
    if (true) {
        std::cout << true;
    }
    else {
        std::cout << false;
    }
}
